### PR TITLE
add list of posts

### PR DESCRIPTION
**Pull request by susickypavel**

### CODE REVIEW

File: components/ListDescription.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const holder = css({
+  height: DESCRIPTION_HEIGHT,
+  flexFlow: 'column',
+  alignItems: 'flex-start',
+  padding: '0 12px',
+})
+
+const postTitle = css({
+  fontSize: 16,
+  color: '#2f3234',
```
Comment (petrbrzek) [inline components/ListDescription.tsx]: Put this color to constants.

File: components/ListDescription.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+})
+
+const postDescription = css({
+  fontSize: 13,
+  color: '#95989a',
```
Comment (petrbrzek) [inline components/ListDescription.tsx]: Put this color to constants.

File: components/ListDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  padding: '0 12px',
+})
+
+const postTitle = css({
+  fontSize: 16,
+  color: '#2f3234',
+})
+
+const postDescription = css({
+  fontSize: 13,
```
Comment (petrbrzek) [inline components/ListDescription.tsx]: There is `FONT_SIZE_4` constant. We should use it, instead of hard-coded values.

File: components/ListItem.tsx [hunk 4]
```diff
@@ -1,12 +1,28 @@
...
-    data: DataObject
+  data: DataObject
 }
 
-const ListItem: React.FunctionComponent<Props> = ({ data }) => (
-  <React.Fragment>{data.id}:{data.name}</React.Fragment>
-);
+const postHolder = css({
+  boxShadow: '0 2px 6px rgba(0, 0, 0, 0.16);',
+  borderRadius: '10px',
```
Comment (petrbrzek) [inline components/ListItem.tsx]: Put this to constants.

File: components/ListItem.tsx [hunk 5]
```diff
@@ -1,12 +1,28 @@
...
+  margin: 9,
```
Comment (petrbrzek) [inline components/ListItem.tsx]: We should use `MAIN_PADDING_1` to be concise. Also, 9 is weird, so no problem to round it to 10.

File: components/ListThumbnail.tsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+const ListThumbnail: React.FunctionComponent<Props> = ({ src }) => {
+  return (
+    <div
+      css={css({
+        height: THUMBNAIL_HEIGHT,
+        background: `no-repeat center url(${src})`,
+        backgroundSize: 'cover',
+        borderRadius: '10px 10px 0px 0px',
```
Comment (petrbrzek) [inline components/ListThumbnail.tsx]: This should be filed by a constant.

**Review by petrbrzek — CHANGES_REQUESTED** [review]
Also, let's create a new folder `fixtures` in `img` folder, where we can put those random images.

**Comment by petrbrzek** [comment]
And rebase it pls.

File: __tests__/index.spec.tsx [hunk 7]
```diff
@@ -48,8 +49,11 @@ it('should render posts', async () => {
   // are solved:
   // https://github.com/kentcdodds/react-testing-library/pull/216
   // https://github.com/facebook/react/issues/14050
-  render(null as any)
-  await wait(0)
+  // render(null as any)
```
Comment (petrbrzek) [inline __tests__/index.spec.tsx]: If this hack is not needed anymore, could you maybe remove the whole block of code?

File: components/List.tsx [hunk 8]
```diff
@@ -1,23 +1,37 @@
-import * as React from "react"
+import * as React from 'react'
```
Comment (petrbrzek) [inline components/List.tsx]: I think we configured TypeScript to import as ES6 syntax. So `import React from 'react'` should work.

File: components/ListItem.tsx [hunk 9]
```diff
@@ -1,12 +1,33 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core'
 import * as React from 'react'
```
Comment (petrbrzek) [inline components/ListItem.tsx]: This line was here, but can you please edit it to ES6 import? :)

File: components/ListThumbnail.tsx [hunk 10]
```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core'
+import * as React from 'react'
```
Comment (petrbrzek) [inline components/ListThumbnail.tsx]: Again, please edit it to ES6 import

File: lib/constants.ts [hunk 11]
```diff
@@ -13,3 +13,12 @@ export const FONT_SIZE_2 = '1.6rem'
 export const FONT_SIZE_3 = '1.4rem'
 export const FONT_SIZE_4 = '1.3rem'
 export const FONT_SIZE_5 = '1.2rem'
+
+export const POST_TITLE_COLOR = '#2f3234'
+export const POST_DESCRIPTION_COLOR = '#95989a'
+export const POST_BOX_SHADOW = '0 2px 6px rgba(0, 0, 0, 0.16)'
+export const POST_BORDER_RADIUS = '10px'
+export const POST_THUMBNAIL_HEIGHT = '165px'
+export const POST_THUMBNAIL = (src: string) => `url(${src}) no-repeat center`
```
Comment (petrbrzek) [inline lib/constants.ts]: This is not constant anymore and it should not be in this file. We can create `utils.ts` file for these use cases.

File: package.json [hunk 12]
```diff
@@ -21,10 +21,10 @@
     "isomorphic-unfetch": "^3.0.0",
     "mdi-react": "^5.2.0",
     "next": "^7.0.2",
-    "react": "^16.8.0-alpha.1",
+    "react": "^16.8.1",
```
Comment (petrbrzek) [inline package.json:24]: Well this should be separate MR but let's say it's fine for now.

File: pages/index.tsx [hunk 13]
```diff
@@ -1,6 +1,8 @@
 import * as React from 'react'
```
Comment (petrbrzek) [inline pages/index.tsx]: Again edit this to ES6 pls.

File: tslint.json [hunk 14]
```diff
@@ -1,6 +1,11 @@
 {
```
Comment (petrbrzek) [inline tslint.json:1]: The file above `svic.jpg` is not in fixtures folder as well as `placeholder-img.jpg`.

**Review by petrbrzek — CHANGES_REQUESTED** [review]

File: __tests__/index.spec.tsx (hunk 7 above)
Comment (susickypavel) [reply __tests__/index.spec.tsx]: I commented these lines by accident, they are uncommented now

File: components/ListThumbnail.tsx [hunk 15]
```diff
@@ -0,0 +1,27 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core'
+import React from 'react'
+import {
+  POST_THUMBNAIL_HEIGHT,
+  POST_THUMBNAIL_BORDER_RADIUS,
+} from '../lib/constants'
+import { POST_THUMBNAIL } from '../lib/utils'
```
Comment (petrbrzek) [inline components/ListThumbnail.tsx:8]: That uppercase style should be used only for constants. I'll change it, that's okay.
